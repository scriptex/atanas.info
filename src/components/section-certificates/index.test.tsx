import * as React from 'react';
import { render } from '@testing-library/react';

import { SectionCertificates } from '.';

describe('SectionCertificates component', () => {
	it('Should render the SectionCertificates component', () => {
		const { asFragment } = render(<SectionCertificates />);

		expect(asFragment()).toMatchSnapshot();
	});
});
