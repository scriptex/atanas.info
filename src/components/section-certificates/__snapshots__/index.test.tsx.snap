// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionCertificates component Should render the SectionCertificates component 1`] = `
<Component
  className=" fullsize-background"
  hasButton={true}
  hasShell={true}
  id="certificates"
  style={
    Object {
      "backgroundImage": "url(images/temp/desktop.jpg)",
    }
  }
>
  <h1>
    Certificates
  </h1>
  <div
    className="o-grid c-section__body"
  >
    <div
      className="o-grid__item xs-12 sm-6"
      key="0"
    >
      <Component
        className="c-certificate"
        href="cert/html.pdf"
      >
        <img
          alt="HTML"
          loading="lazy"
          src="cert/html.jpg"
          width="600"
        />
      </Component>
    </div>
    <div
      className="o-grid__item xs-12 sm-6"
      key="1"
    >
      <Component
        className="c-certificate"
        href="cert/javascript.pdf"
      >
        <img
          alt="JavaScript"
          loading="lazy"
          src="cert/javascript.jpg"
          width="600"
        />
      </Component>
    </div>
    <div
      className="o-grid__item xs-12 sm-6"
      key="2"
    >
      <Component
        className="c-certificate"
        href="cert/python.pdf"
      >
        <img
          alt="Python"
          loading="lazy"
          src="cert/python.jpg"
          width="600"
        />
      </Component>
    </div>
    <div
      className="o-grid__item xs-12 sm-6"
      key="3"
    >
      <Component
        className="c-certificate"
        href="cert/react.pdf"
      >
        <img
          alt="React Development"
          loading="lazy"
          src="cert/react.jpg"
          width="600"
        />
      </Component>
    </div>
    <div
      className="o-grid__item xs-12 sm-6"
      key="4"
    >
      <Component
        className="c-certificate"
        href="cert/css.pdf"
      >
        <img
          alt="CSS"
          loading="lazy"
          src="cert/css.jpg"
          width="600"
        />
      </Component>
    </div>
    <div
      className="o-grid__item xs-12 sm-6"
      key="5"
    >
      <Component
        className="c-certificate"
        href="cert/deno.pdf"
      >
        <img
          alt="Deno"
          loading="lazy"
          src="cert/deno.jpg"
          width="600"
        />
      </Component>
    </div>
  </div>
</Component>
`;
